src/main/modules/SnippetsManager.js

```diff
@@ -1,6 +1,7 @@
 const chars = require('./chars')
 const NativeKeymap = require('native-keymap')
 const _ = require('lodash')
+const PlatformAware = require('./PlatformAware')
 const Notification = require('./Notification')
 const fixPath = require('fix-path')
```

Comment:
Add an if statement, as fixPath is macOS only. It needs to be added to the constructor as well. Assignee @gtluszcz